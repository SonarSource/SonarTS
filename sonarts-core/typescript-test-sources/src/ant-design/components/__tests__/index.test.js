import * as antd from '..';

describe('antd', () => {
  it('exports modules correctly', () => {
    expect(Object.keys(antd)).toMatchSnapshot();
  });
});
