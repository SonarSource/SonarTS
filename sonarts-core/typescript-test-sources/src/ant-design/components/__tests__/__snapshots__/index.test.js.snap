// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`antd exports modules correctly 1`] = `
Array [
  "Affix",
  "Anchor",
  "AutoComplete",
  "Alert",
  "BackTop",
  "Badge",
  "Breadcrumb",
  "Button",
  "Calendar",
  "Card",
  "Collapse",
  "Carousel",
  "Cascader",
  "Checkbox",
  "Col",
  "DatePicker",
  "Dropdown",
  "Form",
  "Icon",
  "Input",
  "InputNumber",
  "Layout",
  "LocaleProvider",
  "message",
  "Menu",
  "Modal",
  "notification",
  "Pagination",
  "Popconfirm",
  "Popover",
  "Progress",
  "Radio",
  "Rate",
  "Row",
  "Select",
  "Slider",
  "Spin",
  "Steps",
  "Switch",
  "Table",
  "Transfer",
  "Tree",
  "TreeSelect",
  "Tabs",
  "Tag",
  "TimePicker",
  "Timeline",
  "Tooltip",
  "Mention",
  "Upload",
  "version",
]
`;
